src/Particle/DistanceTableAA.cpp

```diff
@@ -37,53 +37,63 @@ DistanceTableData* createDistanceTable(ParticleSet& s, int dt_type)
   int sc=s.Lattice.SuperCellEnum;
   DistanceTableData* dt=0;
   std::ostringstream o;
+  std::ostringstream dbg;
   bool useSoA=(dt_type == DT_SOA || dt_type == DT_SOA_PREFERRED);
-  o << "  Distance table for AA: source/target = " << s.getName() << " useSoA =" << useSoA << "\n";
+  o << "  Distance table for similar particles (A-A):" << std::endl;
+  o << "    source/target: " << s.getName() << std::endl;
+  if (useSoA) {
+    o << "    Using structure-of-arrays (SoA) data layout" << std::endl;
+  } else {
+    o << "    Using array-of-structure (AoS) data layout (less efficient than SoA)" << std::endl;
+  }
+
   if(sc == SUPERCELL_BULK)
   {
     if(s.Lattice.DiagonalOnly)
     {
       if(useSoA)
       {
-        o << "    PBC=bulk Orthorhombic=yes  Using SoaDistanceTableAA<T,D,PPPO> of SoA layout " << PPPO << std::endl;
+        o << "    PBC: bulk   Orthorhombic: yes" << std::endl;
+        dbg << "    Using SoaDistanceTableAA<T,D,PPPO> " << PPPO << std::endl;
```

Comment:
The actually useful info in this line is only PPPO. SoA and particle pair have been printed earlier.
Can we just always print "PBC: bulk   Orthorhombic: yes  Type: PPPO"?

Reply:
Removed the debug output and added the 'Type:' information to each line.